client/src/routes.js

```diff
@@ -215,10 +225,22 @@ const routes = [
     path: "/faq",
     component: Faq,
   },
+  {
+    path: "/inbox",
+    component: Inbox,
+    props: {
+      hideFooter: true,
+    },
```

Comment:
Do we need to add the `loggedInOnly` prop (just like the one above for `edit-notifications`) so that it's only visible when logged in?

Reply:
No, using `loggedInOnly` causes some issues and unwanted redirects. I have other measurements in place to make it only visible to logged in.